### DISCUSSION TITLE
AsyncRedisManager with different connections for sub and pub

### DISCUSSION THREAD

**Discussion by alexander-koval** [miguelgrinberg/python-socketio#1156] [Q&A] [answered]
Hello! I've had a little confuse about your AsyncRedisManager. 
What should I do if I have multiple redis servers. One master and couple of replicas. I want to use replicated Redis for subscription and master for publication.
But your AsyncRedisManager support only one url, so I can't split connections. How can I split it to different urls?

**Comment by miguelgrinberg** [comment, marked as answer]
That workflow is currently not supported, but you can implement your own class, maybe starting from a copy of my class, and adapt it to your needs.
